src/main/java/frc/robot/Robot.java

```diff
@@ -32,17 +45,29 @@
   private final SubsystemManager mSubsystemManager = SubsystemManager.getInstance();
 
   //Subsystems
-  private final Drive mDrive = Drive.getInstance();
   private final VisionManager mVisionManager = VisionManager.getInstance();
   private final Shooter mShooter = Shooter.getInstance();
 
   //Variables from State
 
+  
+  static Potentiometer mPot;
+  public static PotPID mPotHandler;
+  static NetworkTable mTable;
+  private double mInitialYaw = 0.0;
+  private static double targetPos = 50;
```

Comment:
Update name to indicate that this is a static variable.

Reply:
Fix